nss-day-cohort-47/tabloidfullstack-fiddlesticks
# Cc admin priv

# Description
Added functionality where only admins can see Users, tag management and category management, Also added some code to begin implementing activate/deactivate users but it is not finished
Fixes # Fixes overall User access based on type of user
## Type of change

- [x ] New feature (non-breaking change which adds functionality)

# Testing Instructions
In order to use this you will have to do three things which will replace the database
1 Run the 01_Tabloid_Create_DB.sql in a query window
2 Run the 02_Tabloid_Seed_Data.sql in a query window
3 Replace your users firebase UID in the User Profile Database

After these three things are done you should be able to login as and admin and have full access. 
If you log in as an author you will not have access to users or to tag and category management

# Checklist:
- [ x] My code follows the style guidelines of this project
- [x ] I have performed a self-review of my own code
- [x ] My changes generate no new warnings or errors
- [ x] I have added test instructions that prove my fix is effective or that my feature works
